### ISSUE TITLE
<InputArea> - the `focus()` method on the testkit doesn't work

### ISSUE THREAD

**Issue by dorduch** [wix/wix-style-react#5337]
```javascript
.trigger('focus', {}) // works
.focus() // doesn't work
```

**Comment by shlomitc** [comment]
Which testkit are we talking about? enzyme/protractor/puppeteer?

**Comment by dorduch** [comment]
enzyme, sorry

**Comment by shlomitc** [comment]
So this was really interesting actually, but I managed to make it work.
Dispatching DOM events (like focus) only works when the DOM container is added to the document.

```jsx
describe('InputArea', () => {
  let container;

  beforeEach(() => {
    container = document.createElement('div');
    document.body.appendChild(container);
  });

  afterEach(() => {
    document.body.removeChild(container);
    container = null;
  });

  it('should trigger focus', () => {
    const handleFocusStub = jest.fn();

    // Mount the "app" on the document
    const dataHook = 'some-hook';
    const wrapper = mount(
      <div data-hook="i-am-the-app">
        <InputArea dataHook={dataHook} onFocus={handleFocusStub} />
      </div>,
      { attachTo: container }, // commenting this out, will fail the test
    );

    const driver = inputAreaTestkitFactory({
      wrapper,
      dataHook,
    });

    expect(handleFocusStub).not.toHaveBeenCalled();
    driver.focus();
    expect(handleFocusStub).toHaveBeenCalled();
  });
});
```

**Comment by shlomitc** [comment]
Anyway, the "trigger" method is tricky since it's not actually dispatching an event but simulating it with test-utils. It might be wrong to have both but pick the one that solves your case.

So I'm closing this since it's a general React constraint and not the testkit's
